test/parser.test.js

```diff
@@ -0,0 +1,47 @@
+const assert = require('assert');
+const parser = require('../parser');
+
+suite('Parser', () => {
+
+    test('getModulesWithPathFromRequireOrDefine should return object with module path and name', () => {
```

Comment:
Until I get a better understanding of the code this is the bare minimum/best I can do. Lets get some tests in place then we can add more soon :)